src/main/resources/config/application-dev.yml

```diff
@@ -37,7 +37,7 @@ spring:
       indent-output: true
   datasource:
     type: com.zaxxer.hikari.HikariDataSource
-    url: jdbc:mysql://localhost:3306/oncokb-public?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
+    url: jdbc:mysql://localhost:3306/oncokb-public?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true
```

Comment:
Why are we adding `allowPublicKeyRetrieval`?

Reply:
For some reason my database would not work for some time- it kept throwing an error "public key retrieval is not allowed." I couldn't even run the application so I went online and found that adding this query worked. I can remove it while I fix the other issues.